buildSrc/build.gradle.kts

```diff
@@ -3,5 +3,11 @@ plugins {
 }
 
 repositories {
+    google()
     mavenCentral()
 }
+
+dependencies {
+    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10")
```

Comment:
Can we get the kotlin version from our gradle.properties here?

Reply:
Our versions are defined in kotlin in Versions file. This buildSrc gradle script is resolved earlier than that (obviously).